From: Chow Loong Jin <hyperair@debian.org>
Date: Mon, 19 Mar 2018 02:36:57 +0800
Subject: Fix format-security error from croak call

Bug: https://github.com/prusa3d/Slic3r/pull/997
Bug-Debian: https://bugs.debian.org/892959
Forwarded: yes
---
 xs/xsp/GCode.xsp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/xs/xsp/GCode.xsp b/xs/xsp/GCode.xsp
index 36b4327..c1856cc 100644
--- a/xs/xsp/GCode.xsp
+++ b/xs/xsp/GCode.xsp
@@ -31,7 +31,7 @@
             try {
                 THIS->do_export(print, path, preview_data);
             } catch (std::exception& e) {
-                croak(e.what());
+                croak("%s\n", e.what());
             }
         %};
 
