Description: Support incompatible change in Boost 1.60
Origin: upstream, https://github.com/alexrj/Slic3r/commit/f4a9fa6569722ef530fac995dc66ec8b6a3179ce
Author: Alessandro Ranellucci <aar@cpan.org>
Last-Update: 2016-03-14
--- a/xs/src/libslic3r/Point.hpp
+++ b/xs/src/libslic3r/Point.hpp
@@ -124,11 +124,15 @@
 }
 
 // start Boost
+#include <boost/version.hpp>
 #include <boost/polygon/polygon.hpp>
 namespace boost { namespace polygon {
     template <>
     struct geometry_concept<coord_t> { typedef coordinate_concept type; };
     
+/* Boost.Polygon already defines a specialization for coordinate_traits<long> as of 1.60:
+   https://github.com/boostorg/polygon/commit/0ac7230dd1f8f34cb12b86c8bb121ae86d3d9b97 */
+#if BOOST_VERSION < 106000
     template <>
     struct coordinate_traits<coord_t> {
         typedef coord_t coordinate_type;
@@ -138,6 +142,7 @@
         typedef long long coordinate_difference;
         typedef long double coordinate_distance;
     };
+#endif
 
     template <>
     struct geometry_concept<Point> { typedef point_concept type; };

