From: Chow Loong Jin <hyperair@debian.org>
Date: Mon, 21 Jan 2019 18:56:10 +0800
Subject: Install XS files into /usr/lib/slic3r-prusa3d

---
 xs/CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

Index: slic3r-prusa/xs/CMakeLists.txt
===================================================================
--- slic3r-prusa.orig/xs/CMakeLists.txt	2019-08-13 20:26:27.000120088 +0800
+++ slic3r-prusa/xs/CMakeLists.txt	2019-08-13 20:27:19.772989432 +0800
@@ -196,8 +196,8 @@
 endif()
 
 # Installation
-install(TARGETS XS DESTINATION ${PERL_VENDORARCH}/auto/Slic3r/XS)
-install(FILES lib/Slic3r/XS.pm DESTINATION ${PERL_VENDORLIB}/Slic3r)
+install(TARGETS XS DESTINATION lib/slic3r-prusa3d/auto/Slic3r/XS)
+install(FILES lib/Slic3r/XS.pm DESTINATION lib/slic3r-prusa3d/Slic3r)
 
 # Unit / integration tests
 enable_testing()
