From: Chow Loong Jin <hyperair@debian.org>
Date: Wed, 15 Nov 2023 09:23:14 +0800
Subject: Use system catch2

Forwarded: not-needed
---
 tests/CMakeLists.txt | 10 +---------
 1 file changed, 1 insertion(+), 9 deletions(-)

diff --git a/tests/CMakeLists.txt b/tests/CMakeLists.txt
index d6d16db..fca13a9 100644
--- a/tests/CMakeLists.txt
+++ b/tests/CMakeLists.txt
@@ -4,15 +4,7 @@
 set(TEST_DATA_DIR ${CMAKE_CURRENT_SOURCE_DIR}/data)
 file(TO_NATIVE_PATH "${TEST_DATA_DIR}" TEST_DATA_DIR)
 
-add_library(Catch2 INTERFACE)
-list (APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake/modules/Catch2)
-target_include_directories(Catch2 INTERFACE ${CMAKE_CURRENT_LIST_DIR})
-add_library(Catch2::Catch2 ALIAS Catch2)
-if (APPLE)
-	# OSX builds targeting OSX 10.9 do not support new std::uncought_exception()
-	# see https://github.com/catchorg/Catch2/issues/1218
-	target_compile_definitions(Catch2 INTERFACE -DCATCH_CONFIG_NO_CPP17_UNCAUGHT_EXCEPTIONS)
-endif()
+find_package(Catch2 REQUIRED)
 include(Catch)
 
 set(CATCH_EXTRA_ARGS "" CACHE STRING "Extra arguments for catch2 test suites.")
